tests/fsyacc/Test1/test1.input3.tokens.bsl

```diff
@@ -0,0 +1,22 @@
+tokenize - getting one token
+tokenize - got LET, now at char 0
```

Comment:
We expect that both `let` from input3 and `LeT` form input4  will result in LET token.